service/src/main/java/uk/gov/mca/beacons/api/beaconuse/domain/BeaconUse.java

```diff
@@ -345,4 +345,8 @@ public String getUseType() {
       ? String.format("%s (%s)", activityName, purpose.name())
       : activityName;
   }
+
+  public void setRandomId(BeaconUseId id) {
```

Comment:
I added it so we could unit test the mapping beacon uses logic. BeaconUseMapper requires the use ID to be not null, and I thought it would be unwise to expose the use ID through an ordinary setter method, so I added this. I thought the consumer of this API would be aware of what they're doing because of the method name so it was ok. Do you want me to remove this?